src/tests/ftest/util/general_utils.py

```diff
@@ -1213,8 +1214,8 @@ def change_file_owner(hosts, filename, owner, group, timeout=15, verbose=True,
 
     """
     return run_command(
-        "{} chown {}:{} {}".format(
-            get_clush_command(hosts, "-S -v", sudo), owner, group, filename),
+        "{} {} {}".format(
+            get_clush_command(hosts, "-S -v", sudo), get_chown_command(owner, group), filename),
```

Comment:
Better to pass filename to `get_chown_command`?
```suggestion
            get_clush_command(hosts, "-S -v", sudo), get_chown_command(owner, group, filename)),
```